README.md

```diff
@@ -726,6 +722,100 @@ Traverse into the "ml_training" folder
   > - If persisted model **xgitguard\output\public\_\*xgg\*.pickle** is not present in the output folder, then use feature engineered data to create a model and persist it.
   > - By default, when feature engineered data collected in Public mode not available, then model creation will be using enterprise-based engineered data.
 
+## Custom Keyword Scan
+
+- Traverse into the `custom-keyword-search` script folder
+
+  ```
+  cd custom-keyword-search
+  ```
+
+#### Running Enterprise Keyword Search
+
+#### Enterprise Custom Keyword Search Process
+
+Please add the required keywords to be searched into config/enterprise_keywords.csv
+
+```
+# Run with given configs,
+python enterprise_keyword_search.py
+```
+
+##### Command to Run Enterprise Scanner for targeted organization
+
+```
+# Run Run for targeted org,
+python enterprise_keyword_search.py -o org_name             #Ex: python enterprise_keyword_search.py -o test_ccs
+```
+
+##### Command to Run Enterprise Scanner for targeted repo
+
+```
+# Run Run for targeted repo,
+python enterprise_keyword_search.py -r org_name/repo_name         #Ex: python enterprise_keyword_search.py -r test_ccs/ccs_repo_1
+```
+
+##### Command-Line Arguments for Enterprsie keyword Scanner
+
+```
+Run usage:
+enterprise_keyword_search.py [-h] [-e Enterprise Keywords]  [-o org_name] [-r repo_name] [-l Logger Level] [-c Console Logging]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -e Enterprise Keywords, --enterprise_keywords Enterprise Keywords
```

Comment:
Add this as optional to search with keywords in cmd line

Reply:
updated